packages/web-components/src/player-component/player.class.ts

```diff
@@ -680,7 +683,7 @@ export class PlayerWrapper {
         const MAX_LATENCY_WINDOW = 3; // in seconds
         this._driftCorrectionTimer = window.setInterval(async () => {
             const video = this.player.getMediaElement() as HTMLMediaElement;
-            if (video && !video.paused && this.isLive && !this.player.isBuffering()) {
+            if (video && !video.paused && this._rtspPlayback && !this.player.isBuffering()) {
```

Comment:
Don't forget to port back to ShakaRTSP so that the verbatim-consumption commit (where we remove code from widgets and instead call that exact same code in ShakaRTSP) is actually verbatim.